ejemplo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <section class="main-container">
+    <img src="phone.png" alt="foto"  />
+    <div class="vertical-separator"> </div>
+    <img src="tcg_logo.png" alt="foto"  />		
+  </section>
+  <div class="mountain-image"></div>
```

Comment:
this is style related so it would be awesome not to contaminate the structure of the page with unnecessary divs. check what are the ```::before``` and ```after``` pseudo elements for. You should use a ```::before``` for this.